assets/xml/overlays/ovl_Boss_05.xml

```diff
@@ -0,0 +1,7 @@
+
+<Root>
```

Comment:
I'll be pedantic and ask if you could remove that first empty line

Reply:
Works for me, I'm the one who is normally pedantic lol